MzTab_MTD

```diff
@@ -168,10 +227,322 @@ CREATE TABLE "MzTab"."PSM Section"
+    spectra_ref text[] COLLATE pg_catalog."default" NOT NULL,
+    identification_method text COLLATE pg_catalog."default" NOT NULL,
+    ms_level text COLLATE pg_catalog."default" NOT NULL,
+    id_confidence_measure_number text COLLATE pg_catalog."default",
+    id_confidence_measure double precision,
+    rank integer NOT NULL,
+    CONSTRAINT "Small Molecule Evidence (SME) Section_pkey1" PRIMARY KEY ("SME_ID")
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE "MzTab-M"."Small Molecule Evidence (SME) Section"
+    OWNER to postgres;
+    
+    
+-- Table: XML.TraML
+
+-- DROP TABLE "XML"."TraML";
+
+CREATE TABLE "XML"."TraML"
+(
+    "id " text COLLATE pg_catalog."default",
+    "version " text COLLATE pg_catalog."default" NOT NULL,
+    "cvList" text[] COLLATE pg_catalog."default",
+    "SourceFileList" text[] COLLATE pg_catalog."default",
+    "ContactList" text[] COLLATE pg_catalog."default",
+    "PublicationList" text[] COLLATE pg_catalog."default",
+    "InstrumentList" text[] COLLATE pg_catalog."default",
+    "SoftwareList" text[] COLLATE pg_catalog."default",
+    "ProteinList" text[] COLLATE pg_catalog."default",
+    "CompoundList" text[] COLLATE pg_catalog."default",
+    "TransitionList" text[] COLLATE pg_catalog."default",
+    "TargetList" text[] COLLATE pg_catalog."default"
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE "XML"."TraML"
+    OWNER to postgres;
+    
+  
+ 
+-- Table: XML.TraML
+
+-- DROP TABLE "XML"."TraML";
+
+CREATE TABLE "XML"."TraML"
+(
+    "GeneName" text COLLATE pg_catalog."default",
+    "ProteinName" text COLLATE pg_catalog."default",
+    "PeptideSequence" text COLLATE pg_catalog."default",
+    "FullPeptideName" text COLLATE pg_catalog."default",
+    transition_group_id text COLLATE pg_catalog."default",
+    transition_name text COLLATE pg_catalog."default",
+    "Annotation" text COLLATE pg_catalog."default",
+    "PrecursorMz" double precision,
+    "MS1 Res" text COLLATE pg_catalog."default",
+    "ProductMz" double precision,
+    "MS2 Res" text COLLATE pg_catalog."default",
+    "Dwell" integer,
+    "Fragmentor" integer,
+    "CollisionEnergy" double precision,
+    "CellAcceleratorVoltage" integer,
+    "LibraryIntensity" double precision,
+    decoy integer,
+    "LabelType" text COLLATE pg_catalog."default",
+    "PrecursorCharge" integer,
+    "FragmentCharge" integer,
+    "FragmentType" text COLLATE pg_catalog."default",
+    "FragmentSeriesNumber" integer,
+    quantifying_transition boolean,
+    identifying_transition boolean,
+    detecting_transition boolean,
+    "Tr_recalibrated" double precision,
+    "RetentionTime" double precision,
+    "PeptideGroupLabel" text COLLATE pg_catalog."default",
+    "UniprotId" text COLLATE pg_catalog."default",
+    "CompoundName" text COLLATE pg_catalog."default",
+    "SMILES" text COLLATE pg_catalog."default",
+    "SumFormula" text COLLATE pg_catalog."default",
+    "mzTab-ID" text COLLATE pg_catalog."default" NOT NULL,
```

Comment:
Is the mzTab-ID text or int?

Reply:
HUPO and PSI use string, text as datatype for mzTab-ID